hydromt/data_adapter/data_adapter.py

```diff
@@ -318,6 +318,7 @@ def _resolve_paths(
         fmt = {}
         # update based on zoomlevel (size = 1)
         if "zoom_level" in keys:
+            zoom_level = 0 if zoom_level is None else zoom_level
```

Comment:
This can also be expressed as `zoom_level = zoom_level or 0` which I personally find a bit more readable, but up to you which you like better, I don't have very strong feelings about it.

Reply:
if find my notation easier to read because `None` is not explicitly mentioned in your notation. But both are fine.

Reply:
True, though I personally find the order of inline if statements in Python somewhat confusing. Again, personal preference, feel free to keep it the way you had it if you want :)